contracts/FTokenStrat.sol

```diff
@@ -81,6 +95,9 @@ contract FTokenStrat is IStrat {
                 )
             );
         }
+
+        rewardpool.exit();
```

Comment:
Seems like we're exiting all our fDai deposits (and FARM rewards) here regardless of divested amount. We should only withdraw the fDai amount needed, then withdraw it to Dai. FARM rewards should be manually claimed by strategist.